### PR TITLE
Modified for feedback during monthly testing

### PR DESCRIPTION
**Pull request by ArchanaShukla**
…'cp-starter-ibpm-employeeonboarding-2-configure', 'cp-starter-ibpm-employeeonboarding-5-run' from QA to Production

### CODE REVIEW

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 1]
```diff
@@ -51,13 +51,15 @@ You will use the **Boosters** to automatically set up the workflow, business rul
 
 1. From your global account page, choose the **Boosters** from left-hand navigation. Among the available options, click **Start** of **Set up account for Workflow Management**.
 
-    !![Start Booster](startrecipe_2.png)
+    !![Start Booster](startrecipe_21.png)
+
+    > If you have multiple sub-accounts, then you will get a wizard to select the sub-account where you want to setup SAP Workflow Management capabilities. In the wizard, select the Subaccount, Org and Space. Click Finish to start the Booster.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:56]: Please fix grammar. It should be "...where you want to set up...".

File: tutorials/cp-starter-ibpm-employeeonboarding-2-configure/cp-starter-ibpm-employeeonboarding-2-configure.md [hunk 2]
```diff
@@ -186,8 +186,8 @@ In this section, you will use **SAP Business Application Studio** to import, con
 
     !![Deploy Workflow](deployworkflow.png)
 
-    >The deployment will fail if you have not logged in to the trial account. Check in the bottom blue bar, if you have already logged in to your trial account. If it does not show organisation-name/space-name then click on the bar and follow the wizard to login to your trial account.
-    !![Login](trial-login.png)
+    >The deployment will fail if you have not logged in to the trial account. Click Function F1 to open the login wizard and follow the wizard to login to your trial account.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-2-configure/cp-starter-ibpm-employeeonboarding-2-configure.md:189]: Please fix grammar. It should be "...follow the wizard to log in..."

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Hello Archana, please fix both grammar mistakes. You may also want to check the other parts of the tutorial. Thank you.

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 1 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md]: Fixed this in all the places

File: tutorials/cp-starter-ibpm-employeeonboarding-2-configure/cp-starter-ibpm-employeeonboarding-2-configure.md (hunk 2 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-2-configure/cp-starter-ibpm-employeeonboarding-2-configure.md]: Fixed this in all the places

**Pull request merged**